### DISCUSSION TITLE
How to show the .kobo folder in KOReader?

### DISCUSSION THREAD

**Discussion by sethhammett** [koreader/koreader#7693] [Q&A] [answered]
Hopefully this should be the last Discussion I have to start :P.
In KOReader I have the show hidden and unsupported files option checked but I still can't see the .kobo folder. Is there any way I can see those or is showing those folders disabled so that people don't brick their readers?

**Comment by Frenzie** [comment, marked as answer]
I suspect the reasoning is more "known useless" than "don't brick" (although those are hardly exclusive), but yes.

https://github.com/koreader/koreader/blob/e6027313e97138c21f014f708b6201bc4a64c350/frontend/ui/widget/filechooser.lua#L29-L51

**Reply by sethhammett** [reply]
Thanks for linking the file path! I was able to edit the code and remove all the excluded directories so now I can see them.

**Reply by NiLuJe** [reply]
FWIW, the reasoning is that anything potentially useful in there would be limited to store-bought Kobo ePubs, and those ought to be DRM'ed, so, unusable in KOReader.
